subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/daemon/DaemonContextParser.java

```diff
@@ -68,8 +69,9 @@ private static DaemonContext parseFrom(String source) {
             String pidStr = matcher.group(4);
             Long pid = pidStr.equals("null") ? null : Long.parseLong(pidStr);
             Integer idleTimeout = Integer.decode(matcher.group(5));
-            List<String> jvmOpts = Lists.newArrayList(Splitter.on(',').split(matcher.group(6)));
-            return new DefaultDaemonContext(uid, new File(javaHome), new File(daemonRegistryDir), pid, idleTimeout, jvmOpts);
+            DaemonParameters.Priority priority = matcher.group(6) == null ? DaemonParameters.Priority.NORMAL : DaemonParameters.Priority.valueOf(matcher.group(6).substring(",priority=".length()));
+            List<String> jvmOpts = Lists.newArrayList(Splitter.on(',').split(matcher.group(7)));
```

Comment:
Is this always group 7 even if `(,priority=[^\\n]+)?` is not present?

Reply:
yep